src/main/java/com/arpnetworking/kairosdb/HistogramDataPointV2Factory.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019 Inscope Metrics, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.kairosdb;
+
+import com.arpnetworking.kairosdb.proto.v2.DataPointV2;
+import com.google.common.collect.Maps;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import org.kairosdb.core.DataPoint;
+import org.kairosdb.core.datapoints.DataPointFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.DataInput;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Optional;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+
+/**
+ * Factory that creates {@link HistogramDataPointV1Impl}.
+ *
+ * @author Brandon Arp (brandon dot arp at smartsheet dot com)
+ */
+public class HistogramDataPointV2Factory implements DataPointFactory {
+    /**
+     * Name of the Data Store Type.
+     */
+    public static final String DST = "kairos_histogram_v2_proto";
```

Comment:
Does this really need to be public? Why not have a static accessor (there is already a non-static one)?